analyzers/tests/SonarAnalyzer.UnitTest/TestCases/WebConfig/DatabasePasswordsShouldBeSecure/Values/Web.config

```diff
@@ -0,0 +1,9 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+  <connectionStrings>
+    <add name="myConnection1" connectionString="Server=myServerAddress;Database=myDataBase;User Id=myUsername;Password=" /> <!-- Noncompliant {{Use a secure password when connecting to this database.}} -->
```

Comment:
You can also add a test case with the `clear` node.

`<clear />`

[Docs](https://docs.microsoft.com/en-us/dotnet/framework/data/adonet/connection-strings-and-configuration-files#the-connectionstrings-section)

Reply:
I added it, however we completely ignore it